tests/unit/test_salesforce.py

```diff
@@ -77,7 +77,10 @@ def test_download_with_query(salesforce):
 
 def test_to_df(salesforce):
     df = salesforce.to_df(table=TABLE_TO_DOWNLOAD)
+    print(len(df.values))
     assert df.empty == False
+    assert len(df.columns) == 98
+    assert len(df.values) >= 1000
```

Comment:
Seems like a lot of rows for testing purposes, is it possible to limit this? I think we don't need more than say, 3 rows?

Reply:
[✅ Improved test_to_df](https://github.com/dyvenia/viadot/pull/629/commits/5adfbe2b9b0de1c1cfc632b19bfc3849512b3f98)